pandas/tests/indexes/multi/test_nested_list_data.py

```diff
@@ -0,0 +1,42 @@
+import pandas as pd
+import pandas._testing as tm
+import numpy as np
+import pytest
+
+
+'''
+This code is meant to test the fix implemented for issue #14467.
+https://github.com/pandas-dev/pandas/issues/14467
+'''
+
+class TestNestedListDataMultiIndex:

```

Comment:
Instead of creating a new file, could you put these tests in an existing file like `pandas/tests/frame/test_constructor.py`